config/locales/en.yml

```diff
@@ -3226,14 +3226,13 @@ en:
             </ul>
           public_service_reveal_header: What service qualifies as a public safety employee?
           public_service_reveal_html: |
-            <p>To qualify, you must be a retired:</p>
+            <p>To qualify, you must have been employed by the federal government, Maryland, or a Maryland political subdivision as a:</p>
             <ul>
               <li>Correctional officer, or</li>
               <li>Law enforcement officer, or</li>
               <li>Firefighter, rescue worker, or emergency services personnel</li>
             </ul>
-            <p>This applies to federal employees in the United States, including Maryland and its political subdivisions.</p>
-          retirement_plan_reveal_header: What retirement plans are considered part of an employee retirement system?
+          retirement_plan_reveal_header: What retirement plans are part of an employee retirement system?
```

Comment:
the spanish copy was not updated for this key, was that intentional?

Reply:
i double checked this with Erica & this is correct :)